pkg/backends/ghz/backend.go

```diff
@@ -133,7 +133,8 @@ func (b *Backend) Sync(ctx context.Context, loadTest loadTestV1.LoadTest, report
 	)
 
 	// Create testfile ConfigMap
-	tfCfgMap, err := NewFileConfigMap(loadTestFileConfigMapName, configFileName, loadTest.Spec.TestFile)
+	configFile := fmt.Sprintf("%s.%s", configFileName, guessTypeFromContent(loadTest.Spec.TestFile))
```

Comment:
I think this should be `configFileName`, since it really is just the filename, not the file itself.  
We can remove the const and rename this to `configFileName`.